crates/astria-gossipnet/src/network.rs

```diff
@@ -192,218 +273,8 @@ impl Network {
-
-        let alice_handle = tokio::task::spawn(async move {
-            let topic = Sha256Topic::new(TEST_TOPIC);
-
-            let mut alice = Network::new(None, 0).unwrap();
-            alice.subscribe(&topic);
-
-            let Some(event) = alice.next().await else {
-                panic!("expected stream event");
-            };
-
-            match event {
-                Event::NewListenAddr(addr) => {
-                    println!("Alice listening on {:?}", addr);
-                    bootnode_tx.send(addr).unwrap();
-                }
-                _ => panic!("unexpected event"),
-            };
-
-            loop {
-                let Some(event) = alice.next().await else {
-                    break;
-                };
-
-                match event {
-                    Event::PeerConnected(peer_id) => {
-                        println!("Alice connected to {:?}", peer_id);
-                    }
-                    Event::PeerSubscribed(peer_id, topic_hash) => {
-                        println!("Remote peer {:?} subscribed to {:?}", peer_id, topic_hash);
-                        alice.publish(msg_a.clone(), topic.clone()).await.unwrap();
-                    }
-                    Event::Message(msg) => {
-                        println!("Alice got message: {:?}", msg);
-                        assert_eq!(msg.data, recv_msg_b);
-                        alice_tx.send(()).unwrap();
-                        return;
-                    }
-                    _ => {}
-                }
-            }
-        });
-
-        let bob_handle = tokio::task::spawn(async move {
-            let topic = Sha256Topic::new(TEST_TOPIC);
-
-            let bootnode = bootnode_rx.await.unwrap();
-            let mut bob = Network::new(Some(vec![bootnode.to_string()]), 0).unwrap();
-            bob.subscribe(&topic);
-
-            loop {
-                select! {
-                    event = bob.next() => {
-                        let Some(event) = event else {
-                            continue;
-                        };
-
-                        match event {
-                            Event::PeerConnected(peer_id) => {
-                                println!("Bob connected to {:?}", peer_id);
-                            }
-                            Event::Message(msg) => {
-                                println!("Bob got message: {:?}", msg);
-                                assert_eq!(msg.data, recv_msg_a);
-                                bob.publish(msg_b.clone(), topic.clone()).await.unwrap();
-                            }
-                            _ => {}
-                        }
-                    }
-                    _ = &mut alice_rx => {
-                        return;
-                    }
-                }
-            }
-        });
 
-        let (res_a, res_b) = join!(alice_handle, bob_handle);
-        res_a.unwrap();
-        res_b.unwrap();
+    pub fn peer_count(&self) -> usize {
```

Comment:
Doc comment. Also why not also just call it `num_peers`?

Reply:
arbitrary, I can rename